cmd/storj-sim/network.go

```diff
@@ -25,8 +25,56 @@ import (
 	"storj.io/storj/internal/processgroup"
 )
 
+type peerClass int
+
+func (p peerClass) String() string {
+	return []string{"Satellite", "Version Control", "Boostrap", "Gateway", "Storage Node"}[p]
+}
+
+const (
+	// The following values of peer class, index, and endpoints are used
+	// to create a port with a consistent format for storj-sim services.
+
+	// Peer class
+	satellitePeer peerClass = iota
+	gatewayPeer
+	versioncontrolPeer
+	bootstrapPeer
+	storagenodePeer
+
+	// Index for peers with one instance (i.e. version control and bootstrap)
+	singleIndex = 0
```

Comment:
Let's remove this and use number `0` in the code. The use of `single` here is different than I had in mind.